src/styles/libraries/bootstrap-overrides/_grid.scss

```diff
@@ -74,13 +90,17 @@
   }
 }
 
-.no-gutters-vertical {
-  margin-top: 0;
-  margin-bottom: 0;
+.no-gutters-vertical-deprecated {
```

Comment:
correct me if i'm wrong, but we should be able to actually kill this, since there is now nothing defining no vertical margin/padding then not defining this will have the same effect

Reply:
well, you didn't want to have to nest the gutters-vertical-deprecated, so we made the selectors a little more lax / less specific, which makes the gutters-vertical-deprecated styles cascade to all the children.  So if you want to turn that back off again for subsequent children, then you have to use this new class